ibm/mas_devops/roles/suite_app_install/README.md

```diff
@@ -16,6 +16,15 @@ Role Variables
 - `mas_app_id` Defines the kind of application that is intended for installation such as `assist`, `health`, `iot`, `manage`, `monitor`, `mso`, `predict`, or `safety`
 - `mas_app_upgrade_strategy` Defines the Upgrade strategy for the MAS Application Operator. Default is set to Automatic
 
+### mas_app_plan
+Defines what plan will be used in application install.
+
+- Environment Variable: `MAS_APP_PLAN`
+- Default: `full`
```

Comment:
Default can't be `full` ... for most apps the value here is irrelevent.

implementation (and doc) should be something like

> ### mas_app_plan
> Defines what plan will be used in application install.
> 
> - Optional
> - Environment Variable: `MAS_APP_PLAN`
> - Default: Application-specific, see details below.
> - Application Support:
>     - Optimizer v8.2+: `full` and `limited` are supported, defaults to `full`

Reply:
@durera maybe what you added above is just an example, but Optimizer will start in 8.2 following MSO 8.1 or will start from 8.0? @lee-cotton @yyjia-cn ?